crates/client/src/account/login.rs

```diff
@@ -1,13 +1,60 @@
-use yew::{function_component, html, Html};
-
 use crate::route::{Route, Router};
+use wasm_bindgen::JsCast;
+use web_sys::{console, HtmlInputElement};
+use yew::{function_component, html, Callback, Html};
 
 #[function_component(Login)]
 pub fn login() -> Html {
+    // Handle submit
+    let onsubmit = Callback::from(move |e: yew::events::SubmitEvent| {
+        e.prevent_default(); // Prevent from something that is default
+
+        // Access the DOM element
+        let email_input = web_sys::window()
+            .unwrap()
+            .document()
+            .unwrap()
+            .get_element_by_id("username")
+            .unwrap()
+            .dyn_into::<HtmlInputElement>()
+            .unwrap();
+
+        // Access the DOM element v2 (Too lazy to create a function for that)
+        let password_input = web_sys::window()
```

Comment:
Create a function for these two long and redundant function chains, on chat we talked about directory with helpers, why you don't use that in the end?